setup.py

```diff
@@ -106,6 +106,7 @@
     'Programming Language :: C',
     "Topic :: Multimedia :: Graphics :: Graphics Conversion",
 ]
+INSTALL_REQUIRES = ['Pillow', 'reportlab', 'pycairo']
```

Comment:
It's not a good idea to use pip deps because UniConvertor is not a python lib but rather a standalone app. It may affects on packaging for different systems.

Reply:
IMHO this is good practice, even for applications.

Both python developers and python packages expect `setup.py` to specify the dependencies. Experience python users expect to be able to install using `pip install`, thus dependencies should be specified here.

Reply:
We have described basic deps in readme. But we don't prepare pip package for UniConvertor. We provide package for different distros with distro-specific dependencies:
https://github.com/sk1project/build-utils/blob/master/src/utils/dependencies.py

As you see, installation requires some native packages, not python only. And we prefer using distro-specific packages because they better manage native deps. Providing python only deps may confuse windows and macos users.

Reply:
Other users have other habits. But I have the impression, you are not open for other mindsets. Thus I abstain from stending more time on this pull-request. Bye!

Reply:
@htgoebel setup.py is a sensitive part of our CI/CD pipeline. Are you sure that these changes don't break build on some of ~25 different OSes for which we prepare daily packages? INSTALL_REQUIRES is good for pip installation, but we are not going provide that. Now application build woks fine and we should have a serious reason to change it. Because this can take a lot of time for testing/debugging.

Reply:
Just for example. CI/CD is based on docker images. Pillow and reportlab are not installed into images because they are not needed for build process. INSTALL_REQUIRES may force checking for these python packages. To fix this issue we need reassembling more than 20 docker images and pull them into DockerHub. This is 2-3 working days of devops job.